### PR TITLE
[DOCS] Update README.md for broken links

### PR DESCRIPTION
**Pull request by cnabro**
- Update README.md for broken links in integrations.
  - https://github.com/great-expectations/great_expectations#integrations
---
- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Comment by cnabro** [comment]
@cla-bot check

**Comment by cnabro** [comment]
@Shinnnyshinshin I can't find any links for `Jupyter Notebooks`. Is this part currently not supported? If do not support it, I will remove it.

(https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations/#creating-expectations-in-jupyter-notebooks)

**Comment by Shinnnyshinshin** [comment]
Hi @cnabro thank you for your interest in contributing to GX! We’re hard at work on GX 1.0 at the moment, and have put accepting PRs on hold until its release. We’ll be updating the API and codebase frequently and significantly over the next few months as we work toward 1.0, and we don’t want contributors like you to put in time and effort only to find that we’ve just implemented a breaking change for your work.

We truly value the contributions and engagement of our community—please hold on to your fantastic ideas until after the 1.0 release, when we’ll be excited to resume accepting PRs. In the meantime, we’re looking for suggestions and feedback on how you use GX and what you’d like to see in GX 1.0, so please share your thoughts with us in our Slack and Discourse channels!

**Comment by Shinnnyshinshin** [comment]
> [reatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations/#creating-expectations-in-jupyter-notebooks](https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations/#creating-expectations-in-jupyter-notebooks))

Hi @cnabro thank you for your contribution. You are correct... the Jupyter Notebooks are currently not supported.

File: README.md [hunk 1]
```diff
@@ -472,27 +472,14 @@ Great Expectations works with the tools and systems that you're already using wi
...
 			<td>
 				Plugin
 			</td>
 		</tr>
-		<tr>
-			<td style="text-align: center; height: 40px; background-color: #fff;">
-				<img height="40" src="./docs/readme_assets/jupyter.jpg" />
-			</td>
-			<td style="width: 200px;">
-				<a href="https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations/#creating-expectations-in-jupyter-notebooks">
```
Comment (Shinnnyshinshin) [inline README.md:488]: Jupyter notebooks not supported as part of Tutorials

**Review by Shinnnyshinshin — APPROVED** [review]

**Pull request merged**